internal/graphicsdriver/metal/graphics_darwin.go

```diff
@@ -21,27 +21,27 @@ import (
 	"strings"
 	"unsafe"
 
+	"github.com/ebitengine/purego/objc"
 	"github.com/hajimehoshi/ebiten/v2/internal/graphics"
```

Comment:
Separater external and internal dependencies i.e. add an empty line after `"github.com/ebitengine/purego/objc"`